setup.py

```diff
@@ -97,6 +97,9 @@
     ],
     url="https://github.com/huggingface/trl",
     packages=find_packages(),
+    entry_points={
+        "console_scripts": ["trl=trl.commands.trl_cli:main"],
```

Comment:
maybe renaming `trl_cli` to `cli` would be a bit less redundant?

Reply:
Makes sense, done !